code/modules/projectiles/ammo_datums.dm

```diff
@@ -2758,7 +2758,7 @@
 			continue
 
 		to_chat(boosting_xeno, SPAN_XENOBOLDNOTICE("Your carapace regenerates as the acid hits [target]!"))
-		boosting_xeno.gain_health(30)
+		boosting_xeno.gain_health(50)
```

Comment:
Far too much. This bad boy is healing as many xenos in a 5 range for 50 health. That's a *ton* of potential AOE healing. Let's go with 35.

Reply:
alright, made the change